manifests/q/QuadSpinner/Gaea/1.3.2.7/QuadSpinner.Gaea.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: QuadSpinner.Gaea
+PackageVersion: 1.3.2.7
+InstallerType: nullsoft
+Installers:
+- Architecture: x64
+  InstallerUrl: https://get.gaea.app/Release/Gaea-1.3.2.7.exe
+  InstallerSha256: 301F1E0BAAC96B99A348F38FA9007961C468F6180BC518A9D9A65920C334AFD9
```

```suggestion
  InstallerSha256: 301F1E0BAAC96B99A348F38FA9007961C468F6180BC518A9D9A65920C334AFD9
  ProductCode: Gaea
```
Product code is required as the publisher and version in ARP are both empty